eng/Version.Details.xml

```diff
@@ -91,5 +91,9 @@
       <Uri>https://github.com/dotnet/roslyn</Uri>
       <Sha>ec366687c8caead948e58704ad61ed644eb91b5b</Sha>
     </Dependency>
+    <Dependency Name="optimization.windows_nt-x64.IBC.CoreFx" Version="99.99.99-master-20190228.4">
```

Comment:
I recall seeing a linux package when you did the optimization work, is that another package?

Reply:
This guy: https://github.com/dotnet/corefx/pull/35675/files#diff-157c769233d1597c193c4aedfe05a488L28

Should we also add an entry for it?

Reply:
Oh sorry I see you're using the same version for both, disregard: https://github.com/dotnet/corefx/pull/35675/files#diff-bf787c0cad9f2c704edb4efb5c4d8400R14

Reply:
Currently it's set up to have the same version so I preserved what was there before.